tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -1,4 +1,4 @@
-# pylint: disable=missing-function-docstring, missing-module-docstring, unidiomatic-typecheck/
+# pylint: disable=missing-function-docstring, redefined-builtin, unused-variable, missing-module-docstring, unidiomatic-typecheck/
```

Comment:
We should not disable codacy errors apart from the docstring errors without good reason. Can you explain why this is necessary?
Usually `redefined-builtin` errors can be fixed by using a new name or for imports using `import X as Y`

`unused-variable` errors can be fixed by renaming the variable to `_` as long as it is not in translated code. But I am not sure which line made you decide you needed this

Reply:
i can get ride of this.

Reply:
it seems Pyccel doesn’t like `import X as Y` it raises ` Undefined variable` error,

Reply:
Hmm, to be clear, this should be `from M import X as Y`. I'm sure it works for functions but maybe there is a problem for data types? If so can you create an issue please

Reply:
sure